test/langtools/tools/javac/annotations/typeAnnotations/NoTargetOnTypeParameterDeclaration.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2023, Alphabet LLC. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8303784
+ * @summary no-@Target annotations should be applicable to type parameter declarations
+ * @compile/fail/ref=NoTargetOnTypeParameterDeclaration.out --release 13 -XDrawDiagnostics NoTargetOnTypeParameterDeclaration.java
+ * @compile --release 14 NoTargetOnTypeParameterDeclaration.java
```

Comment:
suggestion: you could add another command to compile the source with no `--release` option so that the behavior is checked for current and future releases too

Reply:
> suggestion: you could add another command to compile the source with no `--release` option so that the behavior is checked for current and future releases too

Yes, I agree that is a good suggestion and our standard testing practice.